ext-stats/merge.hs

```diff
@@ -134,6 +134,12 @@ main = do
   d <- loadD
   writeFile "GHC2021/result.rst" $ toRst d
 
+printVotes :: Int -> Int -> String
+printVotes votes total = printf formatString votes total
+ where formatString
+        | (fromIntegral votes / fromIntegral total :: Double) >= 2/3 = "**%d/%d**"
+        | otherwise                                                  = "%d/%d"
+
```

Comment:
How about this? No division by 0 risk, and more compact (no total, it’s redundan)
```suggestion
printVotes :: Int -> Int -> String
printVotes votes total = printf formatString votes
 where formatString
        | 3 * votes >= 2 * total = "**%d**"
        | otherwise              = "%d"

```